src/main/java/com/ensao/gi5/lint/visitor/MethodLengthVisitor.java

```diff
@@ -0,0 +1,22 @@
+package com.ensao.gi5.lint.visitor;
+
+import com.ensao.gi5.lint.wrapper.RuleCountWrapper;
+import com.github.javaparser.ast.body.MethodDeclaration;
+import com.github.javaparser.ast.visitor.VoidVisitorAdapter;
+
+import java.util.List;
+
+public class MethodLengthVisitor extends VoidVisitorAdapter<List<RuleCountWrapper>> {
+    private List<RuleCountWrapper> methods;
+    public MethodLengthVisitor(List<RuleCountWrapper> methods){
+        this.methods=methods;
+    }
+    @Override
+    public void visit(MethodDeclaration n, List<RuleCountWrapper> arg) {
+        super.visit(n, arg);
```

Comment:
the `Visitor` pattern requires that the call to superior visits to be done at the end